### DISCUSSION TITLE
Is it possible to have per language reviewers?

### DISCUSSION THREAD

**Discussion by tuxayo** [WeblateOrg/weblate#6472] [Q&A] [answered]
It for when people are reviewer in a language but not as fluent in another. They should be able to suggest but not have that be accepted automatically when submitting. (due to their reviewer status)

Though per language reviewers isn't the only solution. Having reviewers still needing validation from another reviewer would also achieve the same effect.

What are the possibilities of Weblate regarding those matters?

**Comment by nijel** [comment, marked as answer]
You can set permissions per language, though it's currently not that straightforward, see https://github.com/WeblateOrg/weblate/issues/6405#issuecomment-901691878
